optuna/storages/_rdb/models.py

```diff
@@ -268,6 +268,15 @@ def count_past_trials(self, session: orm.Session) -> int:
         )
         return trial_count.scalar()
 
+    @classmethod
+    def whether_to_be_killed(
```

Comment:
How about a name that's less communicating how the return should be used but instead what the logic actual defines? E.g. something like
```suggestion
    def is_trial_stale(
```